elks/arch/i86/drivers/block/init.c

```diff
@@ -44,6 +45,34 @@ void device_setup(void)
 
 #ifdef CONFIG_BLK_DEV_RAM
     rd_load();
+#else
```

Comment:
Why in the define for the RAM disk ?

Reply:
You are right --- that is an error on my part.  Thanks!

Reply:
No problem. You could also cleanup 'rd_load', it is empty everywhere, and not consistent to what I am preparing in term of initial filesystem / disk.

Reply:
@mfld-fr: on `rd_load ()`, I think I will prefer to leave the honours to others.  Thank you!

Reply:
As you want...